cloudhub-2/modules/ROOT/pages/ch2-deploy-private-space.adoc

```diff
@@ -144,6 +144,9 @@ include::partial$deployment.adoc[tag=changeAppProperties,leveloffset=+1]
 // == Configure Log Level
 include::partial$deployment.adoc[tag=configureLogging,leveloffset=+1]
 
+// == Enable Distributed Tracing
```

Comment:
I don't follow where Enable Tracing is in the private space page. I see a hyperlink to [Configure logging for the app](https://github.com/mulesoft/docs-hosting/blob/a38d0ee3f983d4d12e3134d3773d94656827ebae/cloudhub-2/modules/ROOT/pages/ch2-deploy-private-space.adoc#configure-logging). This has to be changed to a Hyperlink to "`Configure **Monitoring** for the app`" instead, based on 1st review comment.

Reply:
This will appear as a new heading, separate from the Logging section